pkg/morserino_com/list_com_test.go

```diff
@@ -0,0 +1,299 @@
+package morserino_com
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.bug.st/serial/enumerator"
+)
+
+//
+// ============ Mock definitions ==================
+
+//Happy case mock
+type mockEnumeratePorts_OK struct{}
+
+//Happy case mocked method
+func (e mockEnumeratePorts_OK) GetDetailedPortsList() ([]*enumerator.PortDetails, error) {
+	happyCaseList := []*enumerator.PortDetails{}
+
+	item1 := enumerator.PortDetails{
+		Name:         "COM Port Name",
+		IsUSB:        true,
+		VID:          "XXXX",
+		PID:          "YYYY",
+		SerialNumber: "001",
+	}
+	item2 := enumerator.PortDetails{
+		Name:         "Morserino Port",
+		IsUSB:        true,
+		VID:          "10C4",
+		PID:          "EA60",
+		SerialNumber: "002",
+	}
+	happyCaseList = append(happyCaseList, &item1)
+	happyCaseList = append(happyCaseList, &item2)
+
+	return happyCaseList, nil
```

Comment:
```suggestion
	return []*enumerator.PortDetails{
		{
			Name:         "COM Port Name",
			IsUSB:        true,
			VID:          "XXXX",
			PID:          "YYYY",
			SerialNumber: "001",
		},
		{
			Name:         "Morserino Port",
			IsUSB:        true,
			VID:          "10C4",
			PID:          "EA60",
			SerialNumber: "002",
		},
	}, nil
```

Style nit: you can declare initialized slices using this syntax. No need to use `append`

Reply:
Learned something new. It is indeed more elegant.